### DISCUSSION TITLE
diff are not showing in the UI

### DISCUSSION THREAD

**Discussion by ghiscoding** [GitbookIO/community#598] [2. Published content]
When creating ` ```diff` in GitHub Wikis, it shows the `-` in pink background and `+` in green background (as shown below) but in GitBook it's just regular text that isn't styled at all and it makes it hard to read.

For example

```diff
this.gridOptions = {
- registerExternalResources: [new ExcelExportService()]
+ externalResources: [new ExcelExportService()]
}
```

in GitBook it shows the following

![image](https://github.com/GitbookIO/community/assets/643976/13ced0d1-3a83-47b8-b742-11c80dc3dd91)

which is unstyled and hard to see the real code differences (which is the main reason to use `diff` and is especially useful when creating Migration Guides for developers)

**Comment by aikoii** [comment]
Hey @ghiscoding! The second screenshot doesn't quite look the way we display diffs in a code block. Here's how it should appear in our UI:

<img width="1272" alt="CleanShot 2024-04-29 at 15 01 09@2x" src="https://github.com/GitbookIO/community/assets/106068994/7661af38-1a0f-4819-9ce7-2368a25079f1">

**Comment by ghiscoding** [comment]
hello @aikoii thanks for the reply, I think I didn't explain myself well enough. What you are showing is the document changes (like a Pull Request), however what I'm referencing is the `gitbook.io` website where all the users go and read the final documentations. For example if you go to mine [https://ghiscoding.gitbook.io/angular-slickgrid](https://ghiscoding.gitbook.io/angular-slickgrid/migrations/migration-to-2.x) and you scroll down the page, you will see that there a few `+` and `-` for the user to follow the migration to a newer version, but it's not styled at all as opposed to a Wiki which is styled.

**Reply by aikoii** [reply]
Ah, @ghiscoding, sorry for the confusion! I've logged this internally. I don't have an ETA yet, but I'll keep you posted 🙂

**Reply by ghiscoding** [reply]
Great to hear that will be super useful for migration guides :)

**Comment by aikoii** [comment]
Hey @ghiscoding! This should be fixed now 🤞

**Reply by ghiscoding** [reply]
thanks but taking a look at some of my migration docs, I still see the same as before. I was expecting the green/red colors to show. Do I have to wait couple days?

https://ghiscoding.gitbook.io/angular-slickgrid/migrations/migration-to-8.x

![image](https://github.com/GitbookIO/community/assets/643976/30c56f43-b728-4454-b2fb-a99f3915a0aa)

**Reply by ghiscoding** [reply]
@aikoii so I waited couple days and I still see the same flat and non-styled text in my public pages. My markdown is using ` ```diff` for example

```diff
- <span class="mdi mdi-undo color-primary"></span>
+ <span class="mdi mdi-undo text-color-primary"></span>
```

Are there any option or flag that I need to enable to see the proper diff coloring? 

EDIT

I just found out that it does show when I'm looking at the page from my dashboard `https://app.gitbook.com/` (great)... but it doesn't show in the public page `https://myusername.gitbook.io/` (sad). As you can imagine, I'm much more interested in the public facing one since that's the one everyone uses and it still doesn't show the styling version yet.

Thanks for the help
